### ISSUE TITLE
No search results on searx.me and other instances

### ISSUE THREAD

**Issue by scroom** [searx/searx#1780]
Hallo,
i run my own searx instance (searx.site) and was happy with it for a long time. However, for some time now I have not been getting any search results. I then checked if this is elsewhere and tested e.g. searx.me. But also here I get de facto no more search results displayed. Have the search engine operators changed their settings? (allowed number of requests, permanent blocking of IPs from which many requests come?)

**Comment by ParadiseStudios** [comment]
Same problem for me. My private local instance is almost worthless now as you have to keep clicking Search until it finally returns results. When it does eventually work, the only search results that come back are from Bing. I tried searx.me and it was the same way. I don't know where the log file is on Debian 10, so I can't check it for errors. I get the feeling the search engines are changing things on the backend and a lot of code will have to be rewritten. Sad.

**Comment by unixfox** [comment]
Which engine do you guys use? If that's Google did you update to the latest version/commit of Searx because a big change was made in the code for that engine: https://github.com/asciimoo/searx/pull/1628

**Comment by ParadiseStudios** [comment]
I tried that fix (https://github.com/MarcAbonce/searx/blob/ccaf6ca02c5bdc63f78e01a66429afaa5fb3cb68/searx/engines/google.py) and also this one: https://github.com/asciimoo/searx/blob/8f51430f5c64e23dba0d518064d4972efec7fe83/searx/engines/google.py for Google but it didn't work so I turned Google off.

All the engines that kept timing out I have turned off, so only Bing is on now. Bing's results are lacking compared to the ones Google turned up. I used to use duckduckgo as a replacement for google results but duckduckgo is temporarily blocked at my ISP level it seems now, so I had to turn that one off too.

**Comment by unixfox** [comment]
After playing with the engine settings I came to the conclusion that bing, ddg and yandex are not working. Apart from that Google is working fine (if you aren't blocked by their captcha), yahoo too, startpage also and probably the other engines too.

I think these engines are blocking the requests because sometimes I get results from yandex , ddg and bing due to the fact that on searx.be the requests are rotated between multiple instances (5 in total).

**Comment by return42** [comment]
To prevent Google CAPTCH a filtron filter is needed and bing had a long standig error which will be fixed after PR #1866 is merged .. if you see any other problems with an engine, please open a separate issue ..
